homeassistant/components/crownstone/devices.py

```diff
@@ -0,0 +1,43 @@
+"""Base classes for Crownstone devices."""
+from __future__ import annotations
+
+from crownstone_cloud.cloud_models.crownstones import Crownstone
+
+from homeassistant.const import (
+    ATTR_IDENTIFIERS,
+    ATTR_MANUFACTURER,
+    ATTR_MODEL,
+    ATTR_NAME,
+    ATTR_SW_VERSION,
+)
+from homeassistant.helpers.entity import DeviceInfo
+
+from .const import CROWNSTONE_INCLUDE_TYPES, DOMAIN
+
+
+class CrownstoneDevice:
```

Comment:
If this is part of the Entity interface the class should inherit `Entity` and be named something with `Entity`, not Device.